debian/base/Dockerfile

```diff
@@ -37,11 +40,12 @@ ENV SCALA_VERSION="2.11"
 ENV CONFLUENT_MAJOR_VERSION="3"
 ENV CONFLUENT_MINOR_VERSION="3"
 
-ENV CONFLUENT_PATCH_VERSION="0"
+ENV CONFLUENT_PATCH_VERSION="0~SNAPSHOT"
 ENV CONFLUENT_VERSION="$CONFLUENT_MAJOR_VERSION.$CONFLUENT_MINOR_VERSION.$CONFLUENT_PATCH_VERSION"
 ENV CONFLUENT_DEB_VERSION="1"
 
-ENV KAFKA_VERSION="0.11.0.0-SNAPSHOT"
+# Add ~SNAPSHOT (with tilde) for internal releases
+ENV KAFKA_VERSION="0.11.0.0~SNAPSHOT"
```

Comment:
+1. I wouldn't seriously merge this, it's too awful. We do need to come up with a wrapper to sort out the differences between build types.